fabtests/multinode/include/core.h

```diff
@@ -56,6 +59,12 @@ enum multi_pattern {
 	PATTERN_BROADCAST,
 };
 
+enum multi_scheduler_type {
+	SCHED_PMIX,
+	SCHED_PMI,
+	SCHED_UNDEF,
+};
```

Comment:
Not sure if scheduler is the most suitable term here. Might just call it "pm". Also I would move the undefined item to the beginning and call it "PM_NONE" if it means no PM at all or "PM_UNSPEC" if it means "don't care".